src/object.jl

```diff
@@ -177,7 +178,7 @@ mutable struct PrefetchedDownloadStream{T <: Object} <: IO
         prefetch_multipart_size > 0 || throw(ArgumentError("`prefetch_multipart_size` must be positive, got $prefetch_multipart_size"))
         if size > 0
             for _ in 1:min(Threads.nthreads(), max(1, div(size, io.prefetch_multipart_size)))
-                Threads.@spawn _download_task($io)
+                Threads.@spawn _download_task($io; $kw...)
```

Comment:
Can you do a quick local check to make sure this works? My gut says I don't know if it should be `($io; $(kw...))` or what you have here, but maybe it doesn't make a difference. Can you just double check by printing keyword args in CloudBase or HTTP to make sure things are getting passed through correctly? I'd hate to update things and then discover later that specific tasks weren't getting keywords passed through right.

Reply:
Yeah I wasn't sure as well so I made this little experiment:
```julia
x = [1,2,3]
for _ in 1:5 # works
    Threads.@spawn println($x...)
end

for _ in 1:5 # error
    Threads.@spawn println($(x...))
end

for _ in 1:5 # works
    Threads.@spawn println($(x)...)
end
```